arenadata/Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG sigar=https://ci.arenadata.io/artifactory/ADB/6.7.1_arenadata4/centos/7/comm
 ARG sigar_headers=https://ci.arenadata.io/artifactory/ADB/6.7.1_arenadata4/centos/7/community/x86_64/sigar-headers-1.6.5-163.el7.x86_64.rpm
 
 # Install some basic utilities and build tools
-RUN yum makecache && \
+RUN yum makecache && yum update -y ca-certificates && \
```

Comment:
So, we assume `ca-certificates` already installed. 
Don't know if it makes a big sense, but using `install` instead of `update` covers both cases: when package is already installed and when package needs an update. Should we use `install` here to increase a flexibility?

Reply:
`yum` depends on this packet. So we can ignore a case of absence `ca-certificates`:

```
yum remove ca-certificates
...
--> Finished Dependency Resolution
Error: Trying to remove "systemd", which is protected
Error: Trying to remove "yum", which is protected
```

Reply:
I wonder, should we also [rebuild](https://access.redhat.com/solutions/1549003) the ca database with `update-ca-trust extract`?